pkg/apis/newrelic/v1alpha1/client.go

```diff
@@ -57,6 +59,10 @@ func handleError(rsp *newrelic.Response, err error) error {
 	} else if rsp.StatusCode < 200 || rsp.StatusCode >= 300 {
 		buf := new(bytes.Buffer)
 		buf.ReadFrom(rsp.Body)
+		if rsp.StatusCode == 404 {
```

Comment:
This is a third-party library that's pulled in we should not make changes like this here.  If not it would require us to fork it.